frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/DistributionField.test.tsx

```diff
@@ -15,18 +15,24 @@ import {
   ClusterStatus,
   CuratorCondition,
   DistributionInfo,
+  HostedClusterApiVersion,
+  HostedClusterKind,
   NodePool,
   ResourceAttributes,
 } from '../../../../../resources'
 import { createBrowserHistory } from 'history'
-import { render, waitFor } from '@testing-library/react'
+import { render, waitFor, screen } from '@testing-library/react'
 import * as nock from 'nock'
 import { RecoilRoot } from 'recoil'
 import { ansibleJobState, clusterImageSetsState, nodePoolsState } from '../../../../../atoms'
 import { nockIgnoreApiPaths, nockIgnoreRBAC, nockRBAC } from '../../../../../lib/nock-util'
 import { clickByText, waitForCalled, waitForNock, waitForNotText, waitForText } from '../../../../../lib/test-util'
 import { DistributionField } from './DistributionField'
 import { Router } from 'react-router-dom'
+import { HostedClusterK8sResource } from 'openshift-assisted-ui-lib/cim'
+import userEvent from '@testing-library/user-event'
+
+import '@testing-library/jest-dom'
```

Comment:
Ideally this shouldn't be needed as it's defined in the frontend/src/setupTests.ts, but could be wrong here. If it's required, just ignore my comment ;)

Reply:
You're correct, it wasn't needed :)

Reply:
Nice, thanks Omar!